src/Exerice1.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const Exercise1 = () => {
+  return (
+    <div className="w-full h-screen flex justify-center items-center bg-yellow-400 text-3xl">
+      hellon world
+    </div>
```

Comment:
Try to correct it here, it is supposed to be `hello, world!`,
second, you may take a look at this instruction.
> React. Start by rendering a square with a background color. Inside the square, render "Hello, World!".